pom.xml

```diff
@@ -52,7 +57,17 @@
 			<artifactId>guava</artifactId>
 			<version>28.2-android</version>
 		</dependency>
-	</dependencies>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>8.0.30</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<version>2.7.3</version>
```

Comment:
Let's keep our dependencies consistent
Put this dependency up to other spring-boot-started dependencies and don't use versioning here

Reply:
Fixed